src/main/resources/db/migration/common/V1_106__insert_system_account.sql

```diff
@@ -0,0 +1 @@
+insert into user_account(usr_id,description) values (0,'System');
```

Comment:
This looks like a good idea to me! We probably need to start being stricter in our definitions for `user_account` as it's strange that a lot of the fields don't currently have `NOT NULL` like user_email_address for example